io-engine/src/logger.rs

```diff
@@ -380,6 +391,21 @@ impl LogFormat {
 
         writeln!(writer)
     }
+
+    fn hostname(&self) -> String {
+        if self.show_host {
+            let mut buf = [0u8; 64];
+            let s = match nix::unistd::gethostname(&mut buf) {
```

Comment:
Should we just store the hostname in the LogFormatter? rather than computing it every time? Then this fn would just become something like:
```rust
fn hostname(&self) -> &str { &self.hostname } ?
```

Reply:
fixed that